core/iwasm/libraries/thread-mgr/thread_manager.c

```diff
@@ -733,6 +733,56 @@ wasm_cluster_create_thread(WASMExecEnv *exec_env,
     return -1;
 }
 
+bool
+wasm_cluster_dup_c_api_imports(wasm_module_inst_t module_inst_dst,
+                               wasm_module_inst_t module_inst_src,
+                               wasm_module_t module)
```

Comment:
Had better use `WASMModuleInstanceCommon *` instead of `wasm_module_inst_t`, we normally use the former internally.
And no need to add argument module, we can get it from module_inst->module